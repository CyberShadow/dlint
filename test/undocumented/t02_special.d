// Special language constructs which should not trigger a positive.

// public imports
public import object;

// compiler-generated methods
private struct WithCopyCtor { this(this) { } }
struct bad01
{
	WithCopyCtor c;
}
