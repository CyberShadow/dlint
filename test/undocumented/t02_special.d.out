undocumented/t02_special.d(8): Undocumented public declaration: StructDeclaration `bad01`
undocumented/t02_special.d(10): Undocumented public declaration: VarDeclaration `c`
